src/Orc.Controls/Controls/Callout/Controls/ApplicationPopup.cs

```diff
@@ -0,0 +1,84 @@
+﻿namespace Orc.Controls
+{
+    using System;
+    using System.Windows;
+    using System.Windows.Controls.Primitives;
+    using System.Windows.Interop;
+
+    internal class ApplicationPopup : Popup
+    {
+        private static readonly IntPtr HWND_TOPMOST = new IntPtr(-1);
+        private static readonly IntPtr HWND_NOTOPMOST = new IntPtr(-2);
+        private static readonly IntPtr HWND_TOP = new IntPtr(0);
+        private static readonly IntPtr HWND_BOTTOM = new IntPtr(1);
+
+        private Window _owner;
+
+        public ApplicationPopup()
+        {
+            Loaded += OnApplicationPopupLoaded;
+            Unloaded += OnApplicationPopupUnloaded;
+        }
+
+        private void OnApplicationPopupLoaded(object sender, RoutedEventArgs e)
+        {
+            _owner = Window.GetWindow(this);
+
+            if (_owner is null)
+            {
+                return;
+            }
+
+            _owner.Activated += OnOwnerWindowActivated;
+            _owner.Deactivated += OnOwnerWindowDeactivated;
+        }
+
+        private void OnApplicationPopupUnloaded(object sender, RoutedEventArgs e)
+        {
+            if (_owner is not null)
+            {
+                _owner.Activated -= OnOwnerWindowActivated;
+                _owner.Deactivated -= OnOwnerWindowDeactivated;
+            }
+
+            Loaded -= OnApplicationPopupLoaded;
```

Comment:
This will prevent it from ever being able to load again. Since you are subscribing to the object itself inside the object itself, I think this will not cause any memory leaks (still allows the object to be garbage collected).